control/margins.py

```diff
@@ -338,7 +336,9 @@ def phase_crossover_frequencies(sys):
 
 
 def margin(*args):
-    """Calculate gain and phase margins and associated crossover frequencies
+    """margin(sys)
```

Comment:
Just below (line 345) the name "sysdata" is used; perhaps that should be used here?